BUILD.bazel

```diff
@@ -55,7 +55,7 @@ pybind_extension(
     copts = [
         "-O3",
         "-pthread",
-        "-std=c++11",
+        "-std=c++17",
```

Comment:
Let's not do this. whisper.cpp compiles with C++11. So we should use C++11 here.